Agents/CEAAgent/README.md

```diff
@@ -1,141 +1,195 @@
+
+If terrain data is required, upload the terrain data as raster files via the [stack-data-uploader]. Please ensure that the `terrain.database` and `terrain.table` in [CEAAgentConfig.properties] matches the RDB database and table containing the uploaded terrain data. Otherwise, the agent will adopt the default terrain values.
+
+### 2.4. Stack Environment
+
+The minimal workflow for the CEA agent using the default setup has the following steps:
+
+1. Transfer the `./stack-manager-input-config/cea-agent.json` and the access agent's service config to the `stack-manager/inputs/config/services` directory.
+2. Start the stack by running `./stack.sh start <STACK NAME>` in the [stack-manager] directory. Replace `<STACK NAME>` with your stack name.
+3. Create the CEA RDB and SPARQL endpoint, corresponding with the `config.properties` file.
+4. Set up routing information on the Access Agent's specified SPARQL endpoint. This is not the SPARQL endpoint indicated in `cea.label`.
+5. Upload building geometry and obda using the [stack-data-uploader].
+6. Send the required HTTP request to the agent to execute its tasks. Please scroll down to find out more
+
+If all goes well, the CEA Agent is reachable at `http://localhost:3838/cea-agent`. `3838` is the default port number used by stack manager.
+If another port number was specified, please replace `3838` accordingly.
+
+### 2.5. Debugging
+
+To debug, put `./stack-manager-input-config/cea-agent-debug.json` instead of `./stack-manager-input-config/cea-agent.json`. Ensure that the `Source` option in the `Mounts` tag of [cea-agent-debug.json] points to your required `CEAAgentConfig.properties`.
+
+Run `./stack.sh start <STACK NAME>` in the [stack-manager] directory to start this service. Once available, the debugger port will be available at 5005.
+
+### 2.6. Visualisation
+
+Visualisation of CEA outputs can be achieved with the [TWA Visualisation Framework] and [FeatureInfoAgent]. The [TWA Visualisation Framework] can be deployed according to the [documentation](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-manager#example---including-a-visualisation).
+
+For the [FeatureInfoAgent], the `.sparql` and `.json` files for CEA data is available at [feature-info-agent-input](./feature-info-agent-input). This must be placed inside the `stack-manager/input/data/fia-queries` directory following the [FeatureInfoAgent] instructions.
+
+Once both the visualisation and FeatureInfoAgent is running on the stack, one should see the following CEA results upon clicking a building:
+
+- solar suitable area _[SCALAR]_
+- annual building energy demands _[SCALAR]_
+- annual photovoltaic potentials _[SCALAR]_
+- building demands _[24H TIME SERIES]_
+- photovoltaic potentials _[24H TIME SERIES]_
```

Comment:
Data on PV panels, solar collectors, and PV-thermal collectors are all shown. Might be good to change the phovoltaic potentials here to solar potentials instead so that it refers to all of the possible solar energy generator. Otherwise, no other issues, and good to go.